modules/ROOT/pages/dataweave-types-coercion.adoc

```diff
@@ -131,7 +128,38 @@ This table shows the possible combinations and the properties from the schema th
 | Value can be `milliseconds` or `seconds`. These are used for Number to DateTime conversions.
 |===
 
-// (1) Returns an array with all the values of the object.
+
+== Using a Java Bridge
+
+DataWeave enables you to call any Java static function or constructor by using the Java bridge. This feature is useful when you need to reuse business logic written in Java or to instantiate any Java object that does not have an empty public constructor.
+
+To use a Java bridge, you must transform a fully qualified Java name to a DataWeave one:
+
+. Add the prefix `!java`.
+. Replace the dots in the fully qualified name with `::`.
+
+When invoking the Java function, DataWeave transforms each argument to the expected type in the function parameter.
+
+The following example shows how to create a new instance of a `java.lang.NullPointerException`:
```

Comment:
```suggestion
The following example shows how to create a new instance of `java.lang.NullPointerException`:
```

Reply:
it's more common to leave out the "a" in this case